charger/wattpilot/wattpilot_mapping_gen.go

```diff
@@ -0,0 +1,217 @@
+package wattpilot
```

Comment:
Wofür brauchen wir eigentlich den ganzen Kram? Können wir nciht einfach direkt den nativen Wert nehmen und gut is?

Reply:
Im Prinzip hier gar nicht - ich schiebe das in die Shell Anwendung von mir